tekton/release-cheat-sheet.md

```diff
@@ -95,26 +105,28 @@ the pipelines repo, a terminal window and a text editor.
       create-draft-release
     ```
 
-    4. Watch logs of create-draft-release
+    1. Watch logs of create-draft-release
 
-    5. On successful completion, a URL will be logged. Visit that URL and sort the
+    1. On successful completion, a URL will be logged. Visit that URL and sort the
```

Comment:
I think we do have to manually add Upgrade and deprecation notices, I am not 100% sure though.

Reply:
that's true, I manually added those notices today

Reply:
I'll add this to the list!